src/filters/index.js

```diff
@@ -8,6 +8,14 @@ module.exports = function (fractal) {
         field_value(str) {
             return str;
         },
+        without(element, exclude) {
+            for (var i = 0; i < exclude.length; i++) {
+                if (element.hasOwnProperty(exclude[i])) {
+                    delete element[exclude[i]];
```

Comment:
1. Will this modify the input variable? Only the output variable should be modified.

2. As we're looking for a particular key, why don't you use `if (element.exclude !== undefined)` without a loop?

Reply:
1. Ok, I will clone the element
2. Exclude can be an array with multiple elements if is called like `|without('excl_1', 'excl_2', '...')`. Is it? what the loop works for is to iterate all the elements to be excluded.

Reply:
2. Oh, I see. In that case, the variable name of the argument should be plural though. And `excludes.forEach()` might be more compact?